scripts/extract-release-notes.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import argparse
+import re
+import subprocess
+
+parser = argparse.ArgumentParser()
+parser.add_argument("out_path")
+args = parser.parse_args()
+
```

Comment:
I'd prefer to have this in a main() and to keep import side-effect free. I know it's just a script, but for debugging, some tools etc, it's sometimes nice.

Reply:
OK, done.